_episodes/02-working-with-environments.md

```diff
@@ -486,35 +486,6 @@ For more on modifying your `.condarc` file, see the
 > {: .solution}
 {: .challenge}
 
-
```

Comment:
Yep, keep it focused on Python, most R users should be learning about `renv` (in my view).